media/humans.txt

```diff
@@ -0,0 +1 @@
+This is media folder for Meowth project
```

Comment:
@Orhideous, I nedded something to have `media` folder in git tree.
BTW, check [[this]](http://google.com/humans.txt)